### PR TITLE
Fix vchambers

### PR DESCRIPTION
**Pull request by Gabrielrezende-asc**
With this PR the BSC is defined by the values of the vacuum chamber on mia mib and mip markers. 
IDs vacuum chambers were also changed, now each ID has its own vacuum chamber, correctly updated.

### CODE REVIEW

**Comment by Gabrielrezende-asc** [comment]
# Note: Delta 52 was added to the model.

**Review by fernandohds564 — APPROVED** [review]

File: pymodels/SI_V25_04/lattice.py [hunk 1]
```diff
@@ -803,38 +801,41 @@ def set_vacuum_chamber(
...
     idp = _pyacc_lat.find_indices(the_ring, 'fam_name', 'id_endp')
-    idp_list = []
-    for i in range(len(idp)//2):
-        idp_list.extend(range(idp[2*i], idp[2*i+1]+1))
-    for i in idp_list:
-        e = the_ring[i]
-        e.vchamber = _pyacc_ele.VChamberShape.rectangle
-        e.hmin, e.hmax, e.vmin, e.vmax = idp_vchamber
+    id_end = ida + idb + idp
+    id_end.sort()
```
Comment (murilobalves) [inline pymodels/SI_V25_04/lattice.py:830]: you could save one line of code with ```id_end = (ida + idb + idp).sort()```

**Review by murilobalves — APPROVED** [review]

File: pymodels/SI_V25_04/lattice.py [hunk 2]
```diff
@@ -803,38 +801,41 @@ def set_vacuum_chamber(
...
+    id_init = id_end[0::2]
+    id_end = id_end[1::2]
+    for subsec, (vch_dim, vch_shape) in ids_vchamber.items():
+        ss_idx = int(subsec[2:4]) - 3  # ss 1 and 2 doesn't have id_end markers
```
Comment (xresende) [inline pymodels/SI_V25_04/lattice.py:834]: 'dont have'

File: pymodels/SI_V25_04/lattice.py [hunk 3]
```diff
@@ -365,9 +365,9 @@ def create_lattice(
         L500p, LID3, L500]  # high beta ID straight section (MANACA)
 
     IDB_10 = [
-        L297, L576p, IDQS, L203, IDBPM, L109, IDC1, L218p,
```
Comment (xresende) [inline pymodels/SI_V25_04/lattice.py:368]: by removing IDQS and IDC1 should you not update `get_family_data` ?

Comment (Gabrielrezende-asc) [reply pymodels/SI_V25_04/lattice.py]: I removed DELTA from the lattice and added EPU again. Next week I will have the information about the new position of DELTA correctors, so I will update the lattice again.

File: pymodels/SI_V25_04/lattice.py [hunk 4]
```diff
@@ -761,11 +760,10 @@ def set_vacuum_chamber(
     scrapv_vchamber = [-0.0145, 0.0145, -0.00475, 0.00475]
     scraph_vchamber = [-0.012, 0.012, -0.012, 0.012]
 
-    ida_vchamber = [-0.012, 0.012, -0.004, 0.004]
-    idb_vchamber = [-0.004, 0.004, -0.00225, 0.00225]
-    idp_vchamber = ida_vchamber
-    if optics_mode.startswith('S05'):
-        idp_vchamber = idb_vchamber
+    # BSC v defined by IVU gap + copper foil = 2 mm @ 1 m from ss center
+    # BSC h defined by vaccum chamber - no limitant
```
Comment (xresende) [inline pymodels/SI_V25_04/lattice.py:764]: Suggestion:

BSC v: defined by IVU magnetic gap + copper foil, 2 mm @ 1m from low beta ss center
BSC h: defined by standard circular vacuum chamber of 12-mm diameter (@ xx meters from high beta ss center)

**Review by fernandohds564 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Pull request merged**